pipeline_template.yaml

```diff
@@ -0,0 +1,37 @@
+---
+version:
+
+date:
+
+pipelines:
+  - models:
+    model_name:
```

Comment:
models vs model_name? is model_name supposed to be a list of all the models?

Reply:
Models is a list containing objects with all the attribute below (after the `:`) so `model_name ` is the name of the model listed in `models`